### PR TITLE
Update URLs and paths

### PR DESCRIPTION
**Pull request by faihegberg**
Update URLs and paths following org and repo rename.

### CODE REVIEW

File: .github/workflows/update-snapshot.yml [hunk 1]
```diff
@@ -72,7 +72,7 @@ jobs:
             * Point at new share images URL.
 
       - name: Wait for Preview URL to be live
-        run: npx wait-on -t 120000 https://covid-projections-git-bump-snapshot-${{env.RUN_ID}}-covidactnow.now.sh/compare/ || echo 'Preview URL failed to load.'
+        run: npx wait-on -t 120000 https://covid-act-now-website-git-bump-snapshot-${{env.RUN_ID}}-covidactnow.now.sh/compare/ || echo 'Preview URL failed to load.'
```
Comment (faihegberg) [inline .github/workflows/update-snapshot.yml:75]: @mikelehen Sean and I aren't 100% sure whether preview links such as this are generated with the beginning of the URL being our org or repo name. We assume the standard should be the repo name. Do you have any insight on this?

File: .github/workflows/update-snapshot.yml [hunk 2]
```diff
@@ -94,7 +94,7 @@ jobs:
         with:
           args: |
             PR to update the website to snapshot ${{env.SNAPSHOT_ID}} (with updated map colors and share images) is available.
-            PR: https://github.com/covid-projections/covid-projections/pull/${{env.PULL_REQUEST_NUMBER}}
-            Preview: https://covid-projections-git-bump-snapshot-${{env.RUN_ID}}-covidactnow.now.sh/
-            Compare: https://covid-projections-git-bump-snapshot-${{env.RUN_ID}}-covidactnow.now.sh/internal/compare/
+            PR: https://github.com/act-now-coalition/covid-act-now-website/pull/${{env.PULL_REQUEST_NUMBER}}
+            Preview: https://covid-act-now-website-git-bump-snapshot-${{env.RUN_ID}}-covidactnow.now.sh/
+            Compare: https://covid-act-now-website-git-bump-snapshot-${{env.RUN_ID}}-covidactnow.now.sh/internal/compare/
```
Comment (faihegberg) [inline .github/workflows/update-snapshot.yml:99]: Same here.

File: scripts/what-the-fips/fix.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ const promisify = require('util').promisify;
 const csvParse = promisify(require('csv-parse'));
 
 const POPULATION_CSV_URL =
-  'https://raw.githubusercontent.com/covid-projections/covid-data-model/main/libs/datasets/sources/fips_population.csv';
+  'https://raw.githubusercontent.com/act-now-coalition/covid-data-model/main/libs/datasets/sources/fips_population.csv';
```
Comment (faihegberg) [inline scripts/what-the-fips/fix.js:10]: Currently this leads to a 404, whether we use `covid-projections` or `act-now-coalition` in the URL. I'm going to try and locate this file.

File: scripts/what-the-fips/index.js [hunk 4]
```diff
@@ -7,7 +7,7 @@ const promisify = require('util').promisify;
 const csvParse = promisify(require('csv-parse'));
 
 const POPULATION_CSV_URL =
-  'https://raw.githubusercontent.com/covid-projections/covid-data-model/main/libs/datasets/sources/fips_population.csv';
+  'https://raw.githubusercontent.com/act-now-coalition/covid-data-model/main/libs/datasets/sources/fips_population.csv';
```
Comment (faihegberg) [inline scripts/what-the-fips/index.js:10]: Same here regarding the 404.

File: scripts/what-the-fips/fix.js (hunk 3 above)
Comment (faihegberg) [reply scripts/what-the-fips/fix.js:10]: Found it. It's in `https://raw.githubusercontent.com/act-now-coalition/covid-data-model/main/data/misc/fips_population.csv`. Will update now.

File: .github/workflows/update-snapshot.yml (hunk 1 above)
Comment (mikelehen) [reply .github/workflows/update-snapshot.yml:75]: Looks like the preview URL for this PR is: https://covid-projections-git-faihegberg-update-urls-8976ae-covidactnow.vercel.app/?s=44733217

So I think it must be the vercel project name and we shouldn't change it (unless we also update the vercel name).

File: .github/workflows/update-snapshot.yml (hunk 2 above)
Comment (mikelehen) [reply .github/workflows/update-snapshot.yml:99]: Same answer.  Seems like it should still be "covid-projections" for now.

File: scripts/what-the-fips/fix.js [hunk 5]
```diff
@@ -7,7 +7,7 @@ const promisify = require('util').promisify;
 const csvParse = promisify(require('csv-parse'));
 
 const POPULATION_CSV_URL =
-  'https://raw.githubusercontent.com/covid-projections/covid-data-model/main/libs/datasets/sources/fips_population.csv';
+  'https://raw.githubusercontent.com/act-now-coalition/covid-data-model/main/data/misc/fips_population.csv';
```
Comment (mikelehen) [inline scripts/what-the-fips/fix.js:10]: Thanks for fixing this. Though for context, this was a one-time script to debug some issues that we could probably delete if we wanted. 🤷

**Review by mikelehen — APPROVED** [review]
LGTM other than the vercel preview URLs!

File: scripts/what-the-fips/fix.js (hunk 5 above)
Comment (faihegberg) [reply scripts/what-the-fips/fix.js:10]: Shall we just delete it then?

Comment (mikelehen) [reply scripts/what-the-fips/fix.js:10]: Feel free!

**Pull request merged**